projects/packages/backup/src/js/components/Admin.js

```diff
@@ -355,7 +355,11 @@ const NoBackupCapabilities = () => {
 	);
 	const priceDetails =
 		introOffer?.interval_unit === 'month' && introOffer?.interval_count === 1
-			? __( 'for the first month, billed yearly', 'jetpack-backup-pkg' )
+			? sprintf(
+					// translators: %s is the regular monthly price
+					__( 'trial for the first month, then $%s/month', 'jetpack-backup-pkg' ),
```

Comment:
```suggestion
					__( 'trial for the first month, then $%s /month', 'jetpack-backup-pkg' ),
```

Following what's in Calypso, this one should contain a space before `/month`. I'm wondering if we should also include the `billed yearly` piece:

```suggestion
					__( 'trial for the first month, then $%s /month, billed yearly', 'jetpack-backup-pkg' ),
```